.github/workflows/clean-envs.yml

```diff
@@ -0,0 +1,34 @@
+name: Execute nightly tests
+
+on:
+  schedule:
+    - cron: "00 0 * * 7"
```

Comment:
is there a reason we have `00` doubled here?

Reply:
also that's `7` at 5th position here? From Github Actions docs:
```
┌───────────── minute (0 - 59)
│ ┌───────────── hour (0 - 23)
│ │ ┌───────────── day of the month (1 - 31)
│ │ │ ┌───────────── month (1 - 12 or JAN-DEC)
│ │ │ │ ┌───────────── day of the week (0 - 6 or SUN-SAT)
│ │ │ │ │
│ │ │ │ │
│ │ │ │ │
* * * * *
```
they accept 0 to 6 (see: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule)